components/mintmaker/staging/pipelines-as-code-secret-path.yaml

```diff
@@ -0,0 +1,4 @@
+---
+- op: add
+  path: /spec/dataFrom/0/extract/key
+  value: staging/mintmaker/github-app
```

Comment:
you have your own key in the vault?

Reply:
changed to use the secret from build-service: `staging/build/stone-stage-p01/github-app`.